src/components/Dialog/index.tsx

```diff
@@ -50,8 +50,10 @@ export function Outer({
   onClose,
   nativeOptions,
   testID,
-  Portal = DefaultPortal,
+  Portal: PortalProp,
```

Comment:
Do we still need this?

Reply:
Hailey seemed to want to keep it around, I'd be fine to rm personally

Reply:
Oh yea this is fine to RM now 👍